src/terminal/parser/ft_fuzzer/string_helper.h

```diff
@@ -41,6 +42,10 @@ void TrimLeft(_In_ T& str, _In_ const typename T::value_type ch)
     }
     else
     {
+		// find_first_not_of returns npos when:
+		// 1. str is empty.

```

Comment:
Make sure you're using spaces, not tabs. 4 spaces.

Reply:
fixed.